src/tests/Provider-test.js

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import 'jsdom-global/register'
+import React, { Component } from 'react'
+import test from 'tape'
+import { findRenderedComponentWithType, renderIntoDocument } from 'react-addons-test-utils'
+import connect from '../connect'
+import Provider from '../Provider'
+import { createMockApp } from './helpers'
+
+test('Should Firebase app instance from context', assert => {
```

Comment:
Then we'll have to change this everywhere else we're using the term "Firebase app instance". I think this is clearer in terms of what we're expecting.